### ISSUE TITLE
Optionally disable algorithms in a pipeline

### ISSUE THREAD

**Issue by jrobble** [openmpf/openmpf#1095]
Users have requested the ability to run multiple algorithms on the same motion output. We currently support this by allowing users to create pipelines where the last stage can run multiple actions in parallel.

Users have also requested the ability to:
- Optionally disable which algorithms are run in the final stage.
- Optionally disable previous stages in the pipeline.

The users' motivation is:
- It's easier for them to configure job properties on media then select different pipelines.
- Ease the burden of configuring feed-forward properties.

Basically, users are interested in having one generic pipeline for each media type (image, audio, video, document), that they can configure via job properties.

---

One way to achieve this is to support a `SKIP_ALGORITHMS` job property that can be set to a list of algorithm names. 

For example `SKIP_ALGORITHMS = FACECV, SUBSENSE` would skip the OpenCV face and SUBSENSE algorithms, no matter where the actions appear in the pipeline (even if for some reason they appear multiple times). If the pipeline is `SUBSENSE --> FACECV & DARKNET`, (where `FACECV` and `DARKNET` are run in parallel in the last stage), it will skip the first stage and the `FACECV` part of the second stage. Only `DARKNET` would run on the frames directly from the media.

If `SKIP_ALGORITHMS = SUBSENSE` and the pipeline is `SUBSENSE --> FACECV & DARKNET` then it will skip the first stage and run both `DARKNET` and `FACECV` in the second stage.

If `SKIP_ALGORITHMS` results in skipping all of the algorithms in the pipeline, the job request should result in an error (no job should be created for invalid requests).

**Comment by jrobble** [comment]
Closing in favor of https://github.com/openmpf/openmpf/issues/1428.
